avocado/core/plugins/multiplexer.py

```diff
@@ -78,7 +109,8 @@ def run(self, args):
             variants = multiplexer.multiplex_yamls(multiplex_files,
                                                    args.filter_only,
                                                    args.filter_out,
-                                                   args.debug)
+                                                   args.debug,
+                                                   additional_tree=args.env)
```

Comment:
How about to split part of `multiplex_yamls()` into `merge_yamls()` and then `mutliplex_yamls()`?.